test/recipes/80-test_cms.t

```diff
@@ -1239,6 +1239,22 @@ with({ exit_checker => sub { return shift == 6; } },
             "Check failure during BIO setup with -stream is handled correctly");
     });
 
+# Check that kari encryption with originator does not segfault
+with({ exit_checker => sub { return shift == 3; } },
+    sub {
+        SKIP: {
+          skip "EC is not supported in this build", 1 if $no_ec;
+
+          ok(run(app(['openssl', 'cms', '-encrypt',
```

Comment:
Simply using `ok(!run(app(...))` (without the `exit_checker`) should be sufficient.

Reply:
Anyway I changed it as this is a temporary test anyway - the plan is to support it so this will go at some point hopefully.